resources/publish/htpasswd

```diff
@@ -0,0 +1 @@
+username:$apr1$gwQzvrYX$hLkvErWORju3C1RS2.QrW0
```

Comment:
is the password for the user name stored as private in some place?

Reply:
I'd be more interested to know which command you've used to generate the username token for "username". 😄 

Something like:

```bash
echo -n 'username' | openssl base64
```

Reply:
Good note, I'll add a comment.

> is the password for the user name stored as private in some place?

This line is a pair of the form `${username}:${hashed password}`.
It is unsafe to upload your personal `.htpasswd`, but this username:password combination is just literally `username:password`. This file is used to authenticate in the local Verdaccio instance, it's not used in an actual server or in production.

> I'd be more interested to know which command you've used to generate the username token for "username". smile

Also very good note. The command is:

```sh
htpasswd -c htpasswd username
```

Which prompts for a password. Note that the output will be (probably) be different every time you generate this file, even if you enter the same password.